src/App/analytics.js

```diff
@@ -1,51 +1,76 @@
 import GoogleAnalytics from 'electron-ga';
 import { reaction } from 'mobx';
 
+const CLIENT_ID = 'UA-22170471-17';
```

Comment:
Naming this CLIENT_ID may be confusing.  Google calls it Tracking ID

Reply:
Renamed to `TRACK_ID` 👍